app/charts/line/lines.tsx

```diff
@@ -179,17 +180,23 @@ const Points = ({ dotSize }: { dotSize: LineFields["y"]["showDotsSize"] }) => {
     useChartState() as LinesState;
   const { chartHeight, width } = bounds;
   const dots = useMemo(() => {
-    return chartData.map((d) => {
-      const x = xScale(getX(d));
-      const y = yScale(getY(d) as number);
-      const fill = colors(getSegment(d));
-
-      return {
-        x,
-        y,
-        fill,
-      };
-    });
+    return chartData
+      .map((d) => {
+        const x = xScale(getX(d));
+        const y = yScale(getY(d) as number);
+        const fill = colors(getSegment(d));
+
+        if (y === undefined) {
```

Comment:
Question: I wonder why x and fill cannot be undefined also ? WOuld it make sense to also add them to this condition ?